src/components/rpc_base/test/rpc_base_test.cc

```diff
@@ -191,18 +191,18 @@ TEST(ValidatedTypes, TestOptionalEmptyArray) {
   Optional<Array<Integer<int8_t, 0, 10>, 0, 5> > ai;
   ASSERT_RPCTYPE_VALID(ai);
   ASSERT_FALSE(ai.is_initialized());
-  Json::FastWriter fw;
-  std::string serialized = fw.write(ai.ToJsonValue());
-  ASSERT_EQ(serialized, "[]\n");
+  Json::StreamWriterBuilder fw;
```

Comment:
@ydementieiev rename it please to something more meaningful. Especially consider that it is not fast writer anymore

Reply:
@AByzhynar done in [1345b2e](https://github.com/smartdevicelink/sdl_core/pull/3155/commits/1345b2e4f52415e896a5fa3d232aa391c69751bb)